pkg/apis/apps/types.go

```diff
@@ -157,6 +177,12 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32
+
+	// PersistentVolumeClaimDeletePolicy indicates the PersistentVolumeClaimDeletePolicy that will
```

Comment:
nit: Would this be more clear ? - PersistentVolumeClaimDeletePolicy indicates the **PersistentVolumeClaimDeletePolicy** => PersistentVolumeClaimDeletePolicy indicates the **policy**

Reply:
done